src/Campaigns/Entities/CampaignV2.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Campaigns\Entities;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+
+class CampaignV2 extends AbstractEntity
```

Comment:
A lot of duplicated code here, why don't extend Campaign entity and add the `$banner` property and override `jsonSerialize` and `__construct` methods? I think the only diff between the 2 entityes is `$banner `instead of `$banners`